src/main/java/org/folio/rest/impl/ServicePointApi.java

```diff
@@ -32,14 +33,17 @@ public class ServicePointApi implements org.folio.rest.jaxrs.resource.ServicePoi
   public static final String SERVICE_POINT_CREATE_ERR_MSG_WITHOUT_BEING_PICKUP_LOC =
     "Hold shelf expiry period cannot be specified when service point cannot be used for pickup";
   private static final Logger logger = LogManager.getLogger();
+  private static final String ECS_ROUTING_QUERY_FILTER = "cql.allRecords=1 NOT ecsRequestRouting=true";
```

Comment:
How this CQL query will work? I can't find the `ecsRequestRouting` field in the service point schema, so there is nothing to request for.

Reply:
This PR is for [ecs-tlr-feature](https://github.com/folio-org/mod-inventory-storage/blob/b7f3a16ff1960123cc2a475ab2d0d42c3af7c7c0/ramls/servicepoint.json#L75) branch, which does contain this field